doc/environment.yml

```diff
@@ -41,3 +41,4 @@ dependencies:
       - pooch
       - versioneer
       - ablog
```

Comment:
should be:
- -e ..

Reply:
format was weird but double dashes with a space in between

Reply:
fixed